app/utils/RegexConstants.scala

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
I've added all the regex I think we will need for future pages from other repos alongside the regex used for this pages errors in the form provider to make it easier for people to use these in future. Hence the TODO to remove unused regex once we are done